src/index.js

```diff
@@ -30,9 +30,7 @@ export default class Swup {
 			animationSelector: '[class*="transition-"]',
 			cache: true,
 			containers: ['#swup'],
-			ignoreLink: (el) => {
-				return el.closest('[data-no-swup]');
-			},
+			ignoreVisit: (href, { el }) => el?.closest('[data-no-swup]'),
```

Comment:
Shouldn't we make the second (options) parameter optional here as well?

```js
ignoreVisit: (href, { el } = {}) => el?.closest('[data-no-swup]'),
```

Reply:
Sure, why not. Feel free to push :)